jekyll/_cci2/add-ssh-key.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: classic-docs
+title: "Adding an SSH Key to CircleCI"
+short-title: "Adding an SSH Key"
+description: "How to Add an SSH Key to CircleCI"
+order: 20
+---
+
+*[Basics]({{ site.baseurl }}/2.0/basics/) > Adding an SSH Key*
+
+If deploying to your servers requires SSH access,
+you'll need to add SSH keys to CircleCI.
+
+## Overview
+
+There are two reasons
+to add SSH keys to CircleCI:
+
+1. To check out code from version control systems.
+2. To enable running processes to access other services.
+
+If you are adding an SSH key for the first reason,
+refer to the [GitHub and Bitbucket Integration](https://circleci.com/docs/2.0/gh-bb-integration/#enable-your-project-to-check-out-additional-private-repositories) document. Otherwise, follow the steps below to add an SSH key to your project.
+
+## Steps
+
+1. In the CircleCI application,
+go to your project's settings
+by clicking the gear icon next to your project.
+
+2. In the **Build Settings** section,
+click on **SSH Permissions**.
+
+3. Click the **Add SSH Key** button.
+
+4. In the **Hostname** field,
+enter the key's associated host (e.g. "git.heroku.com").
```

Comment:
(e.g. ...) -> , for example, `git.heroku.com`.

Reply:
Adding to Style Guide as well.